src/indexApi.ts

```diff
@@ -1,18 +1,114 @@
 // BASE SETUP
 // =============================================================================
 
+import { asObject, asString } from 'cleaners'
 import cluster from 'cluster'
+import { formatISO } from 'date-fns'
 import { forkChildren, rebuildCouch } from 'edge-server-tools'
 
 import { config } from './utils/config'
-import { couchSchema } from './utils/couchSchema'
+import { couchSchema, minAmtDbPrefix } from './utils/couchSchema'
+import { mockPlugins } from './utils/mockPlugins'
 
 const express = require('express')
 const http = require('http')
+const nano = require('nano')
+const promisify = require('promisify-node')
+
+const asData = asObject({
+  minAmount: asString
+})
+
+const asValue = asObject({
+  _id: asString,
+  data: asData
+})
+
+const asFulfilledPromise = asObject({
+  status: asString,
+  value: asValue
+})
```

Comment:
Was it an intentional decision to break up the cleaners in this way?  If you want to shorten your code at the expense of some readability you can combine all three like this: 

```
const asFulfilledPromise = asObject({
  status: asString,
  value: asObject({
    _id: asString,
    data: asObject({ minAmount: asString })
  })
})
```

Reply:
Good find. I like how cleaner this is, so I will incorporate your suggestion.